rmgpy/tools/canteramodel.py

```diff
@@ -75,6 +75,14 @@ def __init__(self, reactor_type, reaction_time, mol_frac, T0=None, P0=None, V0=N
 
         self.mol_frac = mol_frac
 
+        if surface_mol_frac:
+            # normalize surface mol fractions
+            if sum(surface_mol_frac.values()) != 1.00:
```

Comment:
Elsewhere in RMG when we normalize the input fractions we raise a warning (I think in input.py or main.py ?). Can we do the same here with logging.warning?